packages/react-icons/template.js

```diff
@@ -0,0 +1,27 @@
+function template(
+	{ template },
+	opts,
+	{ imports, interfaces, componentName, props, jsx, exports }
+) {
+	const plugins = ['jsx', 'typescript']
+	const tpl = template.smart({ plugins })
+
+	componentName.name = componentName.name.substring(3)
+	exports.declaration.name = componentName.name
+
+	return tpl.ast`
+		import * as React from "react"
+		import { JSX } from "react-jsx"
+
+		import { IFluentIconsProps } from '../IFluentIconsProps.types'
+
+		const ${componentName} = (iconProps: IFluentIconsProps, props: React.HTMLAttributes<HTMLElement>) : JSX.Element=> {
+		const { primaryFill = "#212121", className } = iconProps;
```

Comment:
I think the fill needs to default to null for this to work correctly. Many usages won't want to specify a color for each icon at this level. It doesn't account for the separation of the icon as content and the styling as css. We can author the icons with 212121 but there isn't really a default that seems convenient here aside from "currentcolor", which might be undesirable for other uses. The upside here is that null fill everywhere renders as black by default.